src/components/experiment-tracking/metrics-plots/metrics-plots.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+
+import './metrics-plots.css';
+
+const tabLabels = ['Time-series', 'Parallel coordinates'];
+
+const MetricsPlots = () => {
+  const [activeTab, setActiveTab] = useState(tabLabels[0]);
+
+  return (
+    <div className="metrics-plots-wrapper">
+      <div className="kedro chart-types-wrapper">
+        {tabLabels.map((tab) => {
+          return (
+            <div
+              className={classnames('chart-types-wrapper__tab', {
+                'chart-types-wrapper__tab--active': activeTab === tab,
+              })}
+              key={tab}
+              onClick={() => setActiveTab(tab)}
+            >
+              {tab}
+            </div>
+          );
+        })}
+      </div>
+      <div style={{ fontSize: 14, marginTop: 30 }}>
```

Comment:
hey why don't we have this style in the `metrics-plots.scss`?

Reply:
This is just a placeholder `div` that'll be removed when we add the actual metrics plots in here.